libs/npm/core/src/utils/toQueryString.ts

```diff
@@ -5,8 +5,18 @@
  */
 export const toQueryString = (params: object) => {
   if (params === undefined || params === null) return '';
-  return Object.keys(params)
-    .filter((key) => (params as any)[key] !== undefined)
-    .map((key) => `${encodeURIComponent(key)}=${encodeURIComponent((params as any)[key])}`)
-    .join('&');
+  const values = [];
```

Comment:
Correctly return array parameters as separate query param values.  Before it was joining them with a comma.  But this would result in a single value instead of multiple.